### DISCUSSION TITLE
Is it possible to change the color of a checked checkbox to a custom color?

### DISCUSSION THREAD

**Discussion by N1lzh** [mantinedev/mantine#4202] [Q&A]
I am currently using a chechbox to verify the user has read the TOS before authenticating. Now I ran into the following problem:
Although it is possible to change the color of a checked checkbox to some predefined colors, I did not found the option to customise this color with a color hex code or custom theme.colors. 
Is there a way to use custom colors for the checked state of a checkbox?

![image](https://user-images.githubusercontent.com/61562307/236630315-00d83925-562d-476a-997c-10e8e8c76158.png)

I already tried changing the color with the styles property like this:
`<Checkbox styles={{ input: { backgroundColor: theme.colors.accent,  } }} label="Test" />`
but it only changes the background color of the checkbox when it is not checked.

I appreciate any help, thanks in advance.

**Comment by yverby** [comment]
```js
<Checkbox styles={{ input: { '&:checked': { backgroundColor: ... } } }}  />
```
